### PR TITLE
fix(login): try multiple selector for username

### PR DESCRIPTION
**Pull request by millotp**
Using a single selector is not enough sometime, because there can be 2 input text in any order.
We need a list of selector from most precise to very vague and choose the first one to match a single input.
Feel free to update the list ! At least this one works with the problematic website.

### CODE REVIEW

File: public/test-website/login-double-input.html [hunk 1]
```diff
@@ -2,12 +2,13 @@
 <html lang="en">
 
 <body>
-
+  <form method="POST">
     <input type="text" name="search">
```
[the remark this answers is not in the document]
Comment (millotp) [reply public/test-website/login-double-input.html:6]: good idea

File: src/__tests__/login.test.ts [hunk 2]
```diff
@@ -58,12 +58,10 @@ describe('login', () => {
       password: 'paswword',
     });
 
-    expect(res.statusCode).toBe(200);
+    expect(res.statusCode).toBe(500);
```
Comment (shortcuts) [inline src/__tests__/login.test.ts:61]: should it still error with the new selectors? don't we want to handle it?

Comment (millotp) [reply src/__tests__/login.test.ts:61]: yes but it's a different error, the selectors works fine just there are no cookies sent back, which was an untested error so I kill to bird with one stone. I can add a specific test where everything goes well if you want, or more weird input scenarios

Comment (shortcuts) [reply src/__tests__/login.test.ts:61]: oh oki! as you want it's mostly to ensure we did not broke the current behavior

**Review by shortcuts — APPROVED** [review]
Nice!

**Pull request merged**